blazor/predefined-dialogs/getting-started.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: post
+title: Getting Started with Blazor Predefined Dialogs Component | Syncfusion
+description: Checkout and learn about getting started with Blazor Predefined Dialogs component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Predefined Dialogs
+documentation: ug
+---
+
+# Getting Started with Blazor Predefined Dialogs Component
+
+This section briefly explains how to include Blazor Predefined Dialogs component in your Blazor Server App and Blazor WebAssembly App using Visual Studio.
```

Comment:
This section briefly explains how to include the Blazor Predefined Dialogs component in your Blazor Server App and Blazor WebAssembly App using Visual Studio.